docs/en/stack/ml/nlp/ml-nlp-inference.asciidoc

```diff
@@ -197,6 +198,10 @@ POST _reindex
 However, those web log messages are unlikely to contain enough words for the
 model to accurately identify the language.
 
+TIP: When using deployed NLP models, set the reindex `size` option to a value
+smaller than the deployment's queue size. Otherwise, requests might be rejected
```

Comment:
Is this the queue_capacity mentioned in https://www.elastic.co/guide/en/elasticsearch/reference/master/start-trained-model-deployment.html? If so, it might be clearer to use that setting name explicitly. For example:

```suggestion
smaller than the `queue_capacity` for the trained model deployment. Otherwise, requests might be rejected
```